src/i18n/langResources/lang.de.ts

```diff
@@ -27,6 +27,7 @@ export const langDe: ILangResource = {
   tsOfficialPresetMessage: 'Offizielles TS Icon wird',
   jsonOfficialPresetMessage: 'Offizielles JSON Icon wird',
   hideFoldersPresetMessage: 'Ordner Icons werden',
+  foldersAllDefaultIconPresetMessage: 'Ordner Icons Auflegemaske werden',
```

Comment:
So after a lot of pondering I concluded that the English message should be `Specific folder icons will be enabled/disabled`. I'm using `Specific` and not `Special` because those folder icons don't do anything "special". Their function is the same as the others but they are "specific" to their folder naming.

Are we good?